licenserc.toml

```diff
@@ -0,0 +1,29 @@
+additionalHeaders = ["scripts/custom-header-style.toml"]
+
+headerPath = "scripts/license-header.txt"
+
+includes = [
+    "*.c",
+    "*.cc",
+    "*.h",
+    "*.js",
+    "*.pkl",
+    "*.py",
+    "*.rs",
+    "*.sh",
+    "*.swift",
+    "Cargo.toml",
```

Comment:
I believe all of these files are generated.

Also: toml files aren't software; no need to add a license header to them.

```suggestion
```